glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -87,6 +87,10 @@ open class GleanInternalAPI internal constructor () {
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     internal var isMainProcess: Boolean? = null
 
+    // When sending pings to localhost, we're probably in instrumented tests. In that
+    // case pings are to be immediately submitted by the WorkManager,
+    internal var isSendingToLocalhost: Boolean = false
```

Comment:
I worry (a bit) about the name here, since if the user set the endpoint to localhost through the normal means (not `testSetLocalEndpoint`) it wouldn't be true.  Maybe `isSendingToTestEndpoint`?